tests/sklearn/test_dump_onnx.py

```diff
@@ -216,16 +217,12 @@ def test_dump(
 ) {
   %/_operators.0/Gemm_output_0 = Gemm[alpha = 1, beta = 0, transB = 1](%_operators.0.weight_1, %input_0)
```

Comment:
the following is directly linked to the fact that the `str_in_name` from 
```
    if not is_model_class_in_a_list(
        model_class, get_sklearn_tree_models(str_in_class_name="RandomForest")
    ):
```

was not working, and thus made this test skip all tree-based models (instead of just RF models). Not sure when these ONNX changes appeared, but it could come from a long time ago